feature/interests/impl/src/test/kotlin/com/google/samples/apps/nowinandroid/interests/impl/InterestsListDetailScreenTest.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.samples.apps.nowinandroid.ui
+package com.google.samples.apps.nowinandroid.interests.impl
```

Comment:
It does make sense to move the UI test for this screen into this package, I'm just wondering if there was a reason it was in the `ui` package to start with.

Reply:
That's because `InterestsListDetailScreen` used to live in `app:ui` and so were its tests.